site/src/Librecores/ProjectRepoBundle/Resources/views/Project/view.html.twig

```diff
@@ -37,9 +37,12 @@
           {% if metadata.qualityScore.halfStars == 1 %}
             <span class="fa fa-star-half-full" aria-hidden="true"></span>
           {% endif %}
-          {% for _ in range(metadata.qualityScore.fullStars + metadata.qualityScore.halfStars + 1, 5) %}
-            <span class="fa fa-star-o" aria-hidden="true"></span>
-          {% endfor %}
+          {% set stars = metadata.qualityScore.fullStars + metadata.qualityScore.halfStars %}
```

Comment:
Handling of half-stars is still a bit suspicious. If we somehow get 3 half-stars in the input, the output will be messed up in an interesting way. but it fixes the bug for sure

Reply:
Half stars can never be 3, they can either be 0 or 1. There is a `% 2` in the code for calculating half stars